src/api/spec/models/branch_package_spec.rb

```diff
@@ -27,5 +27,50 @@
         expect(Project.where(name: "#{home_project.name}:branches:BaseDistro:Update")).to exist
       end
     end
+
+    context 'project with ImageTemplates attribute' do
+      let(:attribute_type) { AttribType.find_by_namespace_and_name!('OBS', 'ImageTemplates') }
+
+      context 'auto cleanup attribute' do
+        let(:leap_project) { create(:project, name: 'openSUSE_Leap') }
```

Comment:
This line is repeated in https://github.com/openSUSE/open-build-service/pull/2366/commits/6530b74fd706c4f2fdc71a799aaabe9a779e36d5#diff-04c10c334ba89dca2207895de10a6c69R56

We could move it up to avoid repeating it. :wink: